Octokit.Tests/Models/IssueTest.cs

```diff
@@ -257,7 +257,7 @@ public void CreatesAnIssueUpdateRequestObject()
 
             var update = issue.ToUpdate();
 
-            Assert.Null(update.Labels);
+            Assert.NotNull(update.Labels);
```

Comment:
That was actually my  commit I sent over to @maddin2016's branch. Sorry if it's not correct but it seemed to me .ToUpdate() shouldn't clear the labels from an issue?